website_snippet_marginless_gallery/demo/assets.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+    <template id="demo_assets_editor" inherit_id="website.assets_editor">
```

Comment:
The tour in previous versions was useful only for testing.

The current tour you did I think it's actually helpful for users, so please move this view outside of demo data.